api/turing/service/ensembling_job_service_test.go

```diff
@@ -147,9 +146,9 @@ func generateEnsemblingJobFixture(
 		},
 	}
 	if genExpected {
-		value.JobConfig.Spec.Ensembler.Uri = "/home/spark/ensembler"
+		value.JobConfig.Spec.Ensembler.Uri = "/home/spark/artifact/ensembler"
```

Comment:
Should we replace hardcoded strings with a string value composed from variables: i.e 
```go
fmt.Sprintf("%s/%s/%s", service.sparkHome, ..., ...)
```